src/cli/cli_dataset.cpp

```diff
@@ -704,12 +727,16 @@ template <> otError Dataset::Process<Cmd("active")>(Arg aArgs[])
 
     if (aArgs[0].IsEmpty())
     {
-        error = Print(dataset);
+        error = Print(dataset, false);
```

Comment:
Suggest adding a comment to help make the code more readable; `Print(dataset, false)` does not clearly convey what the `false` input does (the reader needs to look up the function or method definition).
```suggestion
      error = Print(dataset, /* aNonsensitiveOnly */ false);
```

An alternative (and perhaps better) way is to define and use an `enum` (which makes it even easier to read).
```c++
enum DatasetFilter : uint8_t
{
      kAllDatasetFields,
      kNonSensitiveFieldsOnly,
};
```

Then the call would be `Print(dataset, kAllDatasetFields)`, which is even better than `Print(dataset, /* aNonsensitiveOnly */ false);`.

Reply:
Done. Added /* aNonsensitiveOnly */ to make the code more readable.